### DISCUSSION TITLE
cannot export to excel

### DISCUSSION THREAD

**Discussion by beekerc** [dbeaver/dbeaver#23214] [General]
running a query against a jdbc connection to postgresql database.  
query results in 114 rows.  
try to export to excel and i get "Invalid max row size set. Cannot be less than the fetchSize."  
i go into windows / preferences / editors / data editors  
ResultSet fetch size is 200 by default.  i try 0, 500 and 10000.  
same result.  
i check the Use SQL to limit fetch size checkbox and try all 4 fetch sizes again.  
same result.  
what else should i be trying?  
i'm really partial to Oracle's SQLDeveloper, but i'm told i can't use that to connect to this database and have to use dbeaver.

**Comment by LonwoLonwo** [comment]
Hello @beekerc 

Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by shoerley** [comment]
As a workaround, you could export to CSV. Then open you CSV file in Excel, and save it as an Excel file. At least, it will allow yo to determine whether their is something wrong with Excel export, or if it is somewhere else.
